lib/fetch/index.js

```diff
@@ -339,9 +337,8 @@ function fetching ({
   // post-redirect start time are the coarsened shared current time given
   // crossOriginIsolatedCapability.
   const currenTime = coarsenedSharedCurrentTime(crossOriginIsolatedCapability)
-  const timingInfo = makeTimingInfo({
-    startTime: currenTime,
-    postRedirectStartTime: currenTime
+  const timingInfo = createOpaqueTimingInfo({
```

Comment:
I'm not sure if this falls under the category of an opaque timing info but it follows the guidelines of having default values and having startTime = postRedirectStartTime